x-pack/plugins/observability_solution/metrics_data_access/common/inventory_models/container/metrics/charts/cpu.ts

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import {
+  CPU_USAGE_LABEL,
+  DEFAULT_XY_FITTING_FUNCTION,
+  DEFAULT_XY_HIDDEN_AXIS_TITLE,
+  DEFAULT_XY_HIDDEN_LEGEND,
+  DEFAULT_XY_YBOUNDS,
+} from '../../../shared/charts/constants';
+import { LensConfigWithId } from '../../../types';
+import { formulas } from '../formulas';
+
+const containerCpuUsageXY: LensConfigWithId = {
```

Comment:
Perhaps it's better to be more specific.

```suggestion
const dockerContainerCpuUsageXY: LensConfigWithId = {
```